<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventNodeWidget</class>
 <widget class="QWidget" name="EventNodeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>209</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EventNodeWidget</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout" stretch="1,1,1">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="date_days_layout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="date_label">
       <property name="styleSheet">
        <string notr="true">font-family: Microsoft Yahei;
font-size: 22px;
color: white;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>M月d日</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="days_left_label">
       <property name="styleSheet">
        <string notr="true">font-family: Microsoft Yahei;
font-size: 20px;
color: white;</string>
       </property>
       <property name="text">
        <string>%1天后</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="icon_layout">
     <item>
      <spacer name="icon_left_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="icon_label">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">* {
  font-family: Symbols Nerd Font;
  font-size: 23px;
  color: white;
}
.important {
  color: #ee8545;
  font-size: 25px;
}</string>
       </property>
       <property name="text">
        <string></string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="icon_right_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="name_label">
     <property name="styleSheet">
      <string notr="true">font-family: Microsoft Yahei;
font-size: 22px;
color: white;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
